.github/workflows/conformance-kind-proxy-embedded.yaml

```diff
@@ -62,6 +68,7 @@ jobs:
             --helm-set-string=kubeProxyReplacement=true \
             --helm-set=loadBalancer.l7.backend=envoy \
             --helm-set=tls.secretsBackend=k8s \
+            --helm-set=tls.secretSync.enabled=${{ matrix.enable-policy-secret-sync }} \
```

Comment:
Do we need to test it this way? I assume we don't need to run all of the connectivity tests to test with secret sync enabled and vice-versa, no?

Reply:
are you suggesting that we can run relevant tests after flag update in the same job?

Reply:
Correct.